File: bench.java
class Bench {
  public static void main(String[] args){
    int numberOfIterations = 10000000; // 10M
    long sum = 0;
    for(int i = 0; i < numberOfIterations; i++) {
      sum = sum + i;
    }
    System.out.println(sum);
  }
}

File: bench.c
#include <stdio.h>

int main() {
  int numberOfIterations = 10000000; // 10M
  long long int sum = 0 ;
  int i = 0;
  while(i< numberOfIterations) {
    sum = sum + i;
    i++;
  }
  printf("%lld\n", sum);
  return 0;
}
